build/helm/keylime/charts/keylime-agent/templates/daemonset.yaml

```
@@ -8,6 +8,9 @@ spec:
   selector:
     matchLabels:
       {{- include "agent.selectorLabels" . | nindent 6 }}
+  {{- if .Values.serviceAccount.create -}}
```

There might also be an indentation issue here:

```suggestion
    {{- if .Values.serviceAccount.create -}}
```